src/main/java/org/seqdoop/hadoop_bam/KeyIgnoringVCFOutputFormat.java

```diff
@@ -97,7 +97,7 @@ public void readHeaderFrom(SeekableStream in) throws IOException {
 		Configuration conf = ctx.getConfiguration();
 		boolean isCompressed = getCompressOutput(ctx);
 		CompressionCodec codec = null;
-		String extension = "";
+		String extension = ".vcf";
```

Comment:
Won't this get replaced by the compressor extension in the next code block anyway? So it will be "file.bgz" not "file.vcf.bgz".